src/routes/menu.routes.ts

```diff
@@ -1,149 +1,152 @@
+   *     tags: [Menus]
+   *     requestBody:
+   *       required: true
+   *       content:
+   *         application/json:
+   *           schema:
+   *             $ref: '#/components/schemas/Menu'
+   *     responses:
+   *       200:
+   *         description: Creates a menu.
+   *         content:
+   *           application/json:
+   *             schema:
+   *               $ref: '#/components/schemas/Menu'
+   *       500:
+   *         description: Some server error
+   * /menus/{id}:
+   *   get:
+   *     summary: Get the menu by id
+   *     tags: [Menus]
+   *     parameters:
+   *       - in: path
+   *         name: id
+   *         schema:
+   *           type: number
+   *         required: true
+   *         description: The menu id
+   *     responses:
+   *       200:
+   *         description: The menu response by id
+   *         contens:
+   *           application/json:
+   *             schema:
+   *               $ref: '#/components/schemas/Menu'
+   *       500:
+   *         description: Cannot find menu with id=<menu.id>
+   *   put:
+   *    summary: Update the menu by the id
+   *    tags: [Menus]
+   *    parameters:
+   *      - in: path
+   *        name: id
+   *        schema:
+   *          type: number
+   *        required: true
+   *        description: The menu id
+   *    requestBody:
+   *      required: true
+   *      content:
+   *        application/json:
+   *          schema:
+   *            $ref: '#/components/schemas/Menu'
+   *    responses:
+   *      200:
+   *        description: The menu was updated
+   *        content:
+   *          application/json:
+   *            schema:
+   *              $ref: '#/components/schemas/Menu'
+   *      500:
+   *        description: Something went wrong while updating menu with id=<menu.id>
+   *   delete:
+   *     summary: Remove the menu by id
+   *     tags: [Menus]
+   *     parameters:
+   *       - in: path
+   *         name: id
+   *         schema:
+   *           type: number
+   *         required: true
+   *         description: The menu id
+   *
+   *     responses:
+   *       200:
+   *         description: Menu was deleted successfully!
+   *       500:
+   *         description: Couldn't delete menu with id=$<menu.id>
+   */
 
-    //Retrive all menus
-    app.get("/menus", findAllMenus);
+  //Retrive all menus
+  app.get("/menus", findAllMenus);
 
-    //Retrieve one menu by id
-    app.get("/menus/:id", findMenuById);
+  //Retrieve one menu by id
+  app.get("/menus/:id", findMenuById);
 
-    //Create one menu
-    app.post("/menus", createMenu)
+  //Create one menu
+  app.post("/menus", createMenu)
 
-    //Update a menu with id
-    app.put("/menus/:id", updateMenu)
+  //Update a menu with id
+  app.put("/menus/:id", updateMenu)
 
-    //Delete a menu with id
-    app.delete("/menus/:id", deleteMenu)
+  //Delete a menu with id
+  app.delete("/menus/:id", deleteMenu)
+  //Delete recipe from the menu
```

Comment:
Here seems like that comment belong one line below and a blank line should be between it and the above route

Reply:
checked